src/main/java/org/folio/rest/impl/UploadBatchVoucherExportHelper.java

```diff
@@ -150,7 +157,7 @@ private Future<Void> updateHolderWithBatchVoucher(BatchVoucherUploadHolder uploa
   private Future<Void> failUploadUpdate(BatchVoucherExport bvExport, Throwable t) {
     if (bvExport != null) {
       if (!CREDENTIALS_NOT_FOUND.equals(t.getMessage())
-                      && !(t instanceof URISyntaxException)) {
+                      && !(t instanceof URISyntaxException || URL_NOT_FOUND_FOR_FTP.equals(t.getMessage()))) {
```

Comment:
need in somehow to simplify this code, propose to delete this if at all

Reply:
Thanks, Removed the code because of being not valid statement